### PR TITLE
Container audit for typography property-values

### PR DESCRIPTION
**Pull request by laras126**
This PR is an implementation of the following idea for typography properties:
- 'font-size',
- 'font-weight',
- 'font-family',
- 'line-height',
- 'letter-spacing',

This PR will add a menu item "Typography" that will contain data from a property values audit for each of these values.

I didn't commit the built wp-admin.html because we won't be able to preview the deploy anyway, but this branch can be tested by pulling down this branch, running the audit, and checking that the Typography menu item is available when viewing public/wp-admin.html.

--

Record of the initial idea:

One idea for the property-values audits is to provide an outer audit that specifies a list of properties, then runs the property-values audit on each of those. The data from property-values can be reformatted or organized in the containing audit as appropriate, for example, if we want to re-order the results to show all counts of the typography properties next to each other.

This might be useful for spacing and positioning, too!

I'm thinking we keep these as separate audits at first – as opposed to creating a generic "group" audit that accepts an argument for properties – because we might want to write a totally new audit for these values. One idea, for typography, is for an audit to show common groupings of declarations, that way we can see where there is consistent type styling.

### CODE REVIEW

**Comment by ryelle** [comment]
I like this idea! I think it makes a lot of sense to have a few grouped audits. This would also let us spin up new ideas as needed, which might make it clear if we need a new, custom audit for something.

> I'm thinking we keep these as separate audits at first – as opposed to creating a generic "group" audit

➕  definitely agree with this. I think this uses the flexibility of property values nicely, and we don't need another abstraction on top.

**Comment by laras126** [comment]
@ryelle This is ready for review!

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "css-audit": "node ./index.js",
     "lint:js": "eslint src",
-    "format:js": "prettier src *.js --write",
+    "format:js": "prettier src/**/*.js *.js --write",
```
Comment (laras126) [inline package.json:9]: I added this because it was processing the CSS file – we can do that, if it seems better.

**Comment by ryelle** [comment]
The typography audits can now be found on the generated report: https://ryelle.github.io/css-audit/public/wp-admin#typography 💯

**Pull request merged**